### PR TITLE
Rs 2022 06 06.1

### PR DESCRIPTION
**Pull request by rschlaikjer**

### CODE REVIEW

File: script/Arcueid/Day 3/03_00_ARC03_5_1.txt [hunk 1]
```diff
@@ -157,32 +162,32 @@ It hurts.
...
 -- Page 256, Offset 4690.
 -- 内臓が焼けているみたいに、痛い。
-It hurts like my insides are burning.
+It stings as though my insides are on fire.
 }
 [sha:9a194bb8f111ebe8ea1fbd948f00dd96d9149cf8]{
 -- Page 256, Offset 4691.
 -- 涙は止まらなくて、体はゴミのように地面に崩れ落ちた。
-I can't stop crying and my body crumples to the ground like a piece of discarded garbage.
+I can't stop crying, and my body crumples to the ground like a piece of discarded garbage.
```
Comment (yk1512) [inline script/Arcueid/Day 3/03_00_ARC03_5_1.txt:170]: wouldn't add a comma here

File: script/Arcueid/Day 3/03_00_ARC03_5_1.txt [hunk 2]
```diff
@@ -222,17 +228,17 @@ I don't understand why.
...
 -- Page 260, Offset 4703.
 -- 現実感がまるでないから。
-Because it doesn't feel real.
+It has to be. None of this feels real.
 }
 [sha:4ef6e012d37f8edc44bfe6a232ee0c8717367303]{
 -- Page 260, Offset 4704.
 -- ほら、いつもみたいに目眩がして、
-I mean, I feel dizzy like I always do.
+I mean think about it. I had one of my usual dizzy spells, and then...
```
Comment (yk1512) [inline script/Arcueid/Day 3/03_00_ARC03_5_1.txt:236]: comma after mean

File: script/Arcueid/Day 3/03_00_ARC03_5_1.txt [hunk 3]
```diff
@@ -312,42 +320,42 @@ This isn't a dream; I'm just lying to myself.
...
 -- Page 266, Offset 4724.
 -- あの『線』が紙を切るようにモノを切断してしまうと分かっていたのなら。
-If I already understood that these 'lines' would let me cut through things like paper,
+I already understood that these 'lines' let me cut through things as easily as slicing paper.
 }
 [sha:835bf45bb5328af2f06fd26cd2adc68a63863190]{
 -- Page 266, Offset 4725.
 -- 遠野志貴は、人間だって簡単にバラしてしまうと理解していた筈なのに。
-then Tohno Shiki should have known that humans can be torn apart just as easily.
+Tohno Shiki should have known that humans could by torn apart just as easily.
```
Comment (yk1512) [inline script/Arcueid/Day 3/03_00_ARC03_5_1.txt:343]: be

File: script/Arcueid/Day 3/03_00_ARC03_5_1.txt [hunk 4]
```diff
@@ -312,42 +320,42 @@ This isn't a dream; I'm just lying to myself.
...
 -- Page 267, Offset 4727.
 -- ……自分が、簡単に何かを殺してしまう危険な人間なら。
-...If I'm a dangerous person who can easily kill something,
+...I can kill things so easily. I'm a dangerous human being.
 }
 [sha:9815145bb18c84e837728fc603d7d149f503f3c9]{
 -- Page 267, Offset 4728.
 -- 俺はこの目を潰すか、誰とも会わないような生活をするべきだったのに。
-I should have blinded myself or lived a life where I would never see another soul.
+I should have blinded myself, or endeavoured to live a life where I never see another soul.
```
Comment (yk1512) [inline script/Arcueid/Day 3/03_00_ARC03_5_1.txt:358]: where I'd never see

File: script/Arcueid/Day 3/03_00_ARC03_6.txt [hunk 5]
```diff
@@ -267,12 +274,12 @@ Even though I was kneeling in a sea of blood and my legs and arms were bright re
 [sha:a0f06abfc1ddee87352410bbf3acb0d500075507]{
 -- Page 291, Offset 4795.
 -- 先ほどまで<魘|うな>されていたようですし、今もお顔の色が優れません」
-You seemed like you were having a terrible dream. Your face is still pale, even now."
+You seemed like you may have been having a nightmare earlier, and your face is still a little pale."
```
Comment (yk1512) [inline script/Arcueid/Day 3/03_00_ARC03_6.txt:277]: I prefer were

File: script/Arcueid/Day 3/03_00_ARC03_6.txt [hunk 6]
```diff
@@ -322,7 +329,9 @@ It was all a bad dream.
 [sha:317fca2529581594423d05ed8b4e96d0b657ec4f]{
 -- Page 293, Offset 4806.
 -- 意味もなく理由もなく、幼い頃に先生と交わした約束を破って、俺があんなコトをするわけがなかったんだ。
-It's not like I'd do something like that without meaning or reason and break a promise I made with Sensei when I was little.
+// 意味 and 理由 repetition spread out a bit
+There's no way I'd senselessly go and break the promise I made to Sensei when I was little.
+ Not without having any reason like that.
```
Comment (yk1512) [inline script/Arcueid/Day 3/03_00_ARC03_6.txt:334]: Not without having even a single reason.

I don't like the 'any reason' phrasing

File: script/Arcueid/Day 3/03_00_ARC03_6.txt [hunk 7]
```diff
@@ -452,37 +464,37 @@ Please tell Kohaku-san I said thanks."
...
 -- Page 305, Offset 4836.
 -- その夢だと思う出来事の実感も、
-The realization that what I was thinking,
+I can't believe the events that took place in that dream―――
 }
 [sha:76c4d4b306874c28a7a30950e4595b54e31cb2ff]{
 -- Page 305, Offset 4837.
 -- アレが、夢なのだという実感も。
-was actually a dream.
+But nor can I believe that it was actually a dream at all.
```
Comment (yk1512) [inline script/Arcueid/Day 3/03_00_ARC03_6.txt:492]: remove the but

File: script/Arcueid/Day 3/03_00_ARC03_6.txt [hunk 8]
```diff
@@ -517,32 +529,33 @@ On the desk in the room is a knife, a memento from my father.
...
 -- 夢以外の何物でもないじゃないか。
-It was nothing more than a dream.
+What else could it have been but a dream.
 }
 [sha:8bf2761d99c8d6f797e98696ea1457ec4db6ba92]{
 -- Page 307, Offset 4847.
 -- 自分自身を誤魔化すよう、そう繰り返して目蓋を閉じる。
-I close my eyelids, repeating the same phrase, as if to deceive myself.
+// Persuade too weak for 誤魔化? Brainwash?
+I close my eyes, still repeating that phrase over an over in my mind, as if to persuade myself.
```
Comment (yk1512) [inline script/Arcueid/Day 3/03_00_ARC03_6.txt:543]: over and over.

I prefer deceive also

**Review by yk1512 — APPROVED** [review]

File: script/Arcueid/Day 3/03_00_ARC03_6.txt [hunk 9]
```diff
@@ -147,22 +148,23 @@ Hisui slightly raises her eyebrows.
 [sha:5845398749ee9544235a829278ab135645cb9e66]{
 -- Page 281, Offset 4771.
 -- 「はい。志貴さまはベンチで休まれており、その後、ご自分の足で屋敷までお帰りになられました」
-"Yes, you were resting on a bench. After my sister found you, you walked back to the residence on your own two feet."
+"Yes. You had taken a moment to rest on the bench, and afterward returned to the mansion on your own two feet."
```
Comment (Avuxo) [inline script/Arcueid/Day 3/03_00_ARC03_6.txt:151]: despite the redundancy of "you" "you" "your", i think it reads cleaner to me as "afterward you returned"

**Review by Avuxo — APPROVED** [review]

File: script/Arcueid/Day 3/03_00_ARC03_5_1.txt [hunk 10]
```diff
@@ -1,12 +1,12 @@
 [sha:8ccdb5687d855724bd27c0292e82ca92f20fc7a6]{
 -- Page 10012, Offset 4659.
 -- ……こんなのは、悪いユメだ。
-...This is a bad dream.
+...This is just a bad dream.
```
[the remark this answers is not in the document]
Comment (rschlaikjer) [reply script/Arcueid/Day 3/03_00_ARC03_5_1.txt:4]: Those files are only accessible on second playthrough, right? Might leave them for now and fix when I am going through Ciel route TLC

**Pull request merged**